src/main/java/com/edmunds/rest/databricks/DatabricksServiceFactory.java

```diff
@@ -64,17 +76,35 @@ public DatabricksServiceFactory(String username, String password, String host, i
    *
    * @param useLegacyAPI425 choose what version of API compatible HttpClient.
    */
+  @Deprecated
   public DatabricksServiceFactory(String username, String password, String host, int maxRetry,
       long retryInterval, boolean useLegacyAPI425) {
     if (useLegacyAPI425) {
-      client2dot0 = new DatabricksRestClientImpl425(username, password, host, "2.0", maxRetry,
-          retryInterval);
+      client2dot0 = DatabricksRestClientImpl425
+          .createClientWithUserPassword(username, password, host, "2.0", maxRetry,
+              retryInterval);
     } else {
-      client2dot0 = new DatabricksRestClientImpl(username, password, host, "2.0", maxRetry,
-          retryInterval);
+      client2dot0 = DatabricksRestClientImpl
+          .createClientWithUserPassword(username, password, host, "2.0", maxRetry,
+              retryInterval);
     }
   }
 
+  /**
+   * Create a databricks service factory using personal token authentication instead.
+   *
+   * @param personalToken your personal token
+   * @param host the databricks host
+   * @param maxRetry the maximum number of retries
+   * @param retryInterval the retry interval between each attempt
+   */
+  @Deprecated
```

Comment:
Maybe useful to leave a [deprecated java doc](https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html#javadoc_tag)?

Reply:
thanks for a pointer, will add